stdlib/_operator.pyi

```diff
@@ -105,22 +107,10 @@ class attrgetter(Generic[_T_co]):
 
 @final
 class itemgetter(Generic[_T_co]):
-    # mypy lacks support for PEP 646 https://github.com/python/mypy/issues/12280
-    # So we have to define all of these overloads to simulate unpacking the arguments
     @overload
-    def __new__(cls, item: _T_co) -> itemgetter[_T_co]: ...
+    def __new__(cls, __item: _T_co) -> itemgetter[_T_co]: ...
     @overload
-    def __new__(cls, item: _T_co, __item2: _T_co) -> itemgetter[tuple[_T_co, _T_co]]: ...
-    @overload
-    def __new__(cls, item: _T_co, __item2: _T_co, __item3: _T_co) -> itemgetter[tuple[_T_co, _T_co, _T_co]]: ...
-    @overload
-    def __new__(
-        cls, item: _T_co, __item2: _T_co, __item3: _T_co, __item4: _T_co
-    ) -> itemgetter[tuple[_T_co, _T_co, _T_co, _T_co]]: ...
-    @overload
-    def __new__(
-        cls, item: _T_co, __item2: _T_co, __item3: _T_co, __item4: _T_co, *items: _T_co
-    ) -> itemgetter[tuple[_T_co, ...]]: ...
+    def __new__(cls, __item1: _T_co, __item2: _T2_co, *items: Unpack[_Ts]) -> itemgetter[tuple[_T_co, _T2_co, Unpack[_Ts]]]: ...
```

Comment:
It's unclear to me here whether `_T_co` is scoped to the function or the class here. I.e., if you do `x = itemgetter(1, 2, 3)`, is the type of `x` `itemgetter[int]` (`_T_co` is solved as being scoped to the class, and `__item1` is an argument of type `int`, so therefore `x` is of type `itemgetter[int]`? Or is the type of `x` `itemgetter[tuple[int, int, int]]` (`_T_co` is solved as being scoped to the method).

It would be better to avoid using `_T_co` in this overload at all (use a different TypeVar instead), so that it is unambiguous whether the typevar is scoped to the function or the class.

Reply:
Good point, I missed that `_T_co` is also used for the class.

Reply:
Good call. I guess covariance is meaningless for the typevars in this case so we can have something like:
```python
def __new__(cls, __item1: _T1, __item2: _T2, *items: Unpack[_Ts]) -> itemgetter[tuple[_T1, _T2, Unpack[_Ts]]]: ...
```
Or am I not understanding the situation of variance?

Reply:
Yep, you're correct — variance is only meaningful for class-scoped typevars, not function-scoped typevars. Your solution looks correct!